### DISCUSSION TITLE
Fresh argument should be protected

### DISCUSSION THREAD

**Discussion by AidasK** [spatie/laravel-health#125] [General]
`?fresh` argument seems dangerous as it's not protected by any secret and might be a weak point for DDOS attack. We should be able to disable it or at least add a secret in configuration like `?fresh=123123`.

**Comment by freekmurze** [comment]
Feel free to PR this, make sure to add tests.

**Comment by eskiesirius** [comment]
> `?fresh` argument seems dangerous as it's not protected by any secret and might be a weak point for DDOS attack. We should be able to disable it or at least add a secret in configuration like `?fresh=123123`.

you can just extend and remove the fresh command
